### ISSUE TITLE
cannot read property '_dropletEditor' of undefined

### ISSUE THREAD

**Issue by kzidane** [cs50/harvard.cs50.droplet#61]
```
droplet.js:514 Uncaught TypeError: Cannot read property '_dropletEditor' of undefined
  at correctItemDisplay (http://localhost:5050/static/plugins/c9.ide.cs50.droplet/droplet.js:514:92)
  at Array.<anonymous> (http://localhost:5050/static/plugins/c9.ide.cs50.droplet/droplet.js:656:33)
  at Editor.EventEmitter._signal (http://localhost:5050/static/lib/ace/lib/ace/lib/event_emitter.js:76:21)
  at Editor.setSession (http://localhost:5050/static/lib/ace/lib/ace/editor.js:387:14)
  at exports.EventEmitter.<anonymous> (http://localhost:5050/static/plugins/c9.ide.ace/ace.js:2607:29)
  at exports.EventEmitter.EventEmitter.emit (http://localhost:5050/static/lib/events.js:89:33)
  at Plugin.unloadDocument (http://localhost:5050/static/plugins/c9.ide.editors/editor.js:25:17)
  at exports.EventEmitter.<anonymous> (http://localhost:5050/static/plugins/c9.ide.editors/editor.js:62:40)
  at exports.EventEmitter.EventEmitter.emit (http://localhost:5050/static/lib/events.js:89:33)
  at Plugin.unload (http://localhost:5050/static/plugins/c9.core/ext.js:511:23)
  at exports.EventEmitter.<anonymous> (http://localhost:5050/static/plugins/c9.ide.editors/tabmanager.js:543:34)
  at exports.EventEmitter.EventEmitter.emit (http://localhost:5050/static/lib/events.js:89:33)
  at exports.EventEmitter.<anonymous> (http://localhost:5050/static/plugins/c9.ide.editors/tab.js:312:17)
  at exports.EventEmitter.EventEmitter.emit (http://localhost:5050/static/lib/events.js:75:28)
  at Plugin.unload (http://localhost:5050/static/plugins/c9.core/ext.js:511:23)
  at http://localhost:5050/static/plugins/c9.ide.editors/pane.js:128:41
```

![droplet-editor-of-undefined](https://user-images.githubusercontent.com/7230211/28725682-352bb8bc-738c-11e7-83b7-da2b116004b5.gif)

**Comment by dlloyd09** [comment]
@dabbler0 I think we'll actually need to fix this one as well, before we can deploy. #60 might be able to wait. Moving this one to "above the line".

**Comment by dabbler0** [comment]
My fix for #59 also resolves this and #60.
